File: 프로그래머스/lv2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    answer=[]
    for a in range(len(arr1)):
        arr=[]
        for c in range(len(arr2[0])):
            arrSum=0
            for b in range(len(arr2)):
                arrSum+=arr1[a][b] * arr2[b][c]
            arr.append(arrSum)
        answer.append(arr)
    return answer

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    const answer = []
    
    for(let i = 0; i < arr1.length; i++) {
        const addAnswer = []
        for(let j = 0; j < arr2[0].length; j++) {
            let sum = 0
            for(let n = 0; n < arr1[0].length; n++) {
                sum += (arr1[i][n] * arr2[n][j])
            }
            addAnswer.push(sum)
        }
        answer.push(addAnswer)
    }
    
    return answer
}
